<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="HOME_IMG">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1041</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#MONITORING {
	background-color: rgb(0, 0, 0);
	border-radius:5px;
}

QPushButton#MONITORING:hover{
 background-color: rgb(0, 255, 0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>HOME_sCRN.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="MONITORING">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#MONITORING {
	background-color: rgb(0, 0, 0);
	border-radius:5px;
}

QPushButton#MONITORING:hover{
 background-color: rgb(0, 255, 0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>1.2 icon1.png</normaloff>1.2 icon1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="VOLUME">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#VOLUME {

	background-color: rgb(0, 0, 0);
 border-radius:5px;
}

QPushButton#VOLUME:hover{
 background-color: rgb(0, 255, 0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>1.3 icon2.png</normaloff>1.3 icon2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="EXIT">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>450</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#EXIT {

	background-color: rgb(0, 0, 0);
 border-radius:5px;
}

QPushButton#EXIT:hover{
 background-color: rgb(0, 255, 0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>1.4 icon3.png</normaloff>1.4 icon3.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="CAM_WINDOW">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>190</y>
      <width>561</width>
      <height>331</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="VOLUME_SLIDER">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>310</y>
      <width>22</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="sliderPosition">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="VOLUME_LEVEL">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
